homeassistant_cli/helper.py

```diff
@@ -83,6 +83,7 @@ def req_raw(
     url = '{}/api/{}'.format(ctx.server, endpoint)
     headers = {
         'Authorization': 'Bearer {}'.format(ctx.token),
+        'x-ha-access': ctx.password,
```

Comment:
this fix is fine but in a few hours I'll hopefully do a merge that removes this "deprecated" req_raw method so don't have to deal with passwords in two places.